Cubical/Relation/Nullary/Properties.agda

```diff
@@ -0,0 +1,152 @@
+{-# OPTIONS --cubical --safe #-}
+{-
+
+Properties of nullary relations, i.e. structures on types.
+
+Includes several results from "Generalizations of Hedberg's Theorem"
+by Altenkirch, Coquand, Escardo, Kraus
```

Comment:
Can you add a link to the paper you read? Was it the published or arxiv version? (sometimes they differ...)

Reply:
I initially read the [version](https://nicolaikraus.github.io/docs/hedberg.pdf) found on Nicolai Kraus' webpage, which refers to [this publication](https://link.springer.com/chapter/10.1007/978-3-642-38946-7_14). On Arxiv is ["Notions of Anonymous Existence in Martin-Löf Type Theory"](https://arxiv.org/abs/1610.03346) with very similar content. Double-check differences, change the name and link to the arxiv version?

Reply:
I'll use the version linked by @martinescardo below